Makefile

```diff
@@ -22,3 +24,14 @@ uninstall:
 .PHONY: clean
 clean:
 	rm -rf .build
+
+# Following https://www.smileykeith.com/2021/03/03/editing-rpaths/
+.PHONY: release
```

Comment:
it's be really nifty if this command took a version tag string as an argument and could run `git tag` as well.

Though maybe we'd still want the ability to `make:release_build` without that functionality. In any case, we can add this later.

Reply:
Yeah, ideally I'd love to do everything from the command line and also release through homebrew but one thing at a time 😁